macros/CompatBeta.ejs

```diff
@@ -456,6 +456,42 @@ function writeNotes(support, browserId) {
   return output;
 }
 
+/**
+ * Formats the `browserNameKey` for display in the UI
+ * @param {String} browserNameKey - The browser name
+ * @returns Correctly formatted browser name
+ */
+function getFormattedBrowserName(browserNameKey) {
```

Comment:
This function is OK for now. In the future, the formatted name should come from the compat data package.
(the export is `browsers.browserNameKey.name`), but we're missing two browsers still (see https://github.com/mdn/browser-compat-data/pull/2690 and https://github.com/mdn/browser-compat-data/pull/1712)

Reply:
Ah yes, that will make this much simpler.